### PR TITLE
Initialize docs for the infrastructure monitoring

### PR DESCRIPTION
**Pull request by dedemorton**

### CODE REVIEW

**Comment by dedemorton** [comment]
@karenzone @tbragin This is a work in progress, but thought I would push my changes early.

I'm running short on time and need to deliver minimal docs to support the beta. Note that Karen is working in parallel on the topics for the Kibana docs. Dan is going to give me access to an instance tomorrow that I can use to (hopefully) get some good screen grabs for the container stuff.

File: docs/en/infraops/infra-ui-intro.asciidoc [hunk 1]
```diff
@@ -0,0 +1,73 @@
+[[infrastructure-ui-overview]]
+[role="x-pack"]
```
Comment (dedemorton) [inline docs/en/infraops/infra-ui-intro.asciidoc]: Fixed the typo here that keeps x-pack badge from showing.

File: docs/en/infraops/infra-ui-intro.asciidoc [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+// and power of the UI. Screen captures with realisitc data (to be added) will
+// help users understand the value.
+
+[float]
+=== Monitor your hosts and containers
+
+You start with an overview of the hosts and containers running in your
+infrastructure. The overview provides a summary of high-level metrics, like CPU
+usage, system load, memory usage, and network traffic, to help you assess the
+overall health of your systems and services.
```
Comment (skh) [inline docs/en/infraops/infra-ui-intro.asciidoc]: Maybe add a note somewhere that the view can be (and has to be) switched between hosts, containers and (k8s) pods by using the links at the top right.

Comment (dedemorton) [reply docs/en/infraops/infra-ui-intro.asciidoc]: I'd like to keep this discussion more conceptual. We'll be adding other topics to Kibana to cover the functioning of the UI. Thanks for your input!

File: docs/en/infraops/installation.asciidoc [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+and enable modules for the metrics you want to collect.
+
+For either approach, you need to enable the following modules to populate the
+{infra-ui} UI with data:
+
+* {metricbeat-ref}/metricbeat-module-system.html[{metricbeat} `system` module]
+* {filebeat-ref}/filebeat-module-system.html[{filebeat} `system` module]
+* {metricbeat-ref}/metricbeat-module-docker.html[{metricbeat} `docker` module]
+* {metricbeat-ref}/metricbeat-module-kubernetes.html[{metricbeat} `kubernetes` module]
+
```
Comment (dedemorton) [inline docs/en/infraops/installation.asciidoc]: I need to incorporate the following feedback from Dan R:

The modules you listed are correct for metricbeat to populate the waffle map in Infrastructure.  I would also say in the docs that from the Infrastructure UI you can drill down to the Logs and Metrics for a Host, Pod, or Container; so add the Filebeat modules for the system and whatever is running (Apache2, Redis, MySQL, etc.)

File: docs/en/infraops/infra-ui-intro.asciidoc [hunk 4]
```diff
@@ -0,0 +1,80 @@
+[[infrastructure-ui-overview]]
+[role="xpack"]
+== {infra-ui} UI
+
+beta[]
+
+After you have infrastructure monitoring <<install-infrastructure-monitoring,up and running>>
+and data is streaming to {es}, use the infrastructure monitoring UI in {kib} to monitor your
```
Comment (dedemorton) [inline docs/en/infraops/infra-ui-intro.asciidoc]: Used the wrong variable here. Should say Infrastructure UI.

Comment (dedemorton) [inline docs/en/infraops/infra-ui-intro.asciidoc]: TODO: Add link to Karen's topics in the Kibana docs when they are available.

File: docs/en/infraops/installation.asciidoc [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+// REVIEWERS: Have I missed anything? Is there more detail required, or is it
+// OK (for beta, at least) to direct users to the Add data tutorials and
+// docs for install/config info? I'm concerned that the container monitoring
+// info might be incomplete.
```
[the remark this answers is not in the document]
Comment (dedemorton) [reply docs/en/infraops/installation.asciidoc]: I'll add some links.

**Comment by tbragin** [comment]
@dedemorton I took a look at the documents at a high level. 

My only comment is that the Logs UI (which is a separate tab in Kibana) is a bit buried from the content perspective. Is there a way to add a very simple page to the structure at a high level covering it, even if it is just the screenshot and duplicate info from [this page](https://inframonitoring-62ede.firebaseapp.com/infrastructure-ui-overview.html#_view_related_logs)?

docs

My assumption is that this book will appear as so in the overall docs structure for 6.5, but that we'll evolve it from there.

screen shot 2018-11-08 at 2 59 09 pm

File: docs/en/infraops/installation.asciidoc (hunk 5 above)
[the remark this answers is not in the document]
Comment (dedemorton) [reply docs/en/infraops/installation.asciidoc]: @weltenwort I've added more detail base don your comment. For processors, I'm not quite sure when users need to add them. Can you please review: https://github.com/elastic/stack-docs/pull/142/commits/86ba462ce952fdd6f9e799bc64c8081aa4acfb7c

**Comment by dedemorton** [comment]
@tbragin Yes, I will make the section shorter and add a link that points to the Logging UI topic.

File: docs/en/infraops/installation.asciidoc [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+* Any other {filebeat-ref}/filebeat-modules.html[{filebeat} modules] needed for
+your environment, such as `apache2`, `redis`, and so on
+* The {filebeat-ref}/filebeat-input-docker.html[docker input in {filebeat}]
+
+
+// REVIEWERS: This language is too wishy washy. Can you help me come up with
+// better language to help users understand when this is required?
+
+You may also need to add processors to your configuration to collect metadata
+about containers or hosts where your services are running:
```
Comment (weltenwort) [inline docs/en/infraops/installation.asciidoc]: This is how I understand it:

* If the user enables the filebeat `docker` input to tail docker logs or the metricbeat docker module, she should also enable the `add_docker_metadata` processor.
* If the user enables the filebeat `docker` input to tail kubernetes logs or the metricbeat kubernetes module, she should also enable the `add_kubernetes_metadata` processor.
* The `add_cloud_metadata` and `add_host_metadata` processors are enabled by default starting with 6.5.

We could structure this section to match the three node types currently supported, e.g.

* If the user wants to collect host system logs and see hosts and their metrics in the Infrastructure UI, she should enable the `system` filebeat and metricbeat modules.
* If the user wants to collect docker container logs and see containers and their metrics in the Infrastructure UI, she should enable the `docker` filebeat input and metricbeat module as well as the `add_docker_metadata`. Autodiscovery can help to automate that.
* If the user wants to collect kubernetes container logs and see pods and their metrics in the Infrastructure UI, she should enable the `docker` filebeat input and `kubernetes` metricbeat module as well as the `add_kubernetes_metadata`. Autodiscovery can help to automate that.

**Pull request merged**